src/js/descript.js

```diff
@@ -99,29 +121,39 @@
         return this;
     };
 
+    /**
+     * Removes specified scripts from the default container.
+     * @param scriptAttributes
+     * @returns {Descript}
+     */
     Descript.prototype.remove = function(scriptAttributes) {
         this._process(scriptAttributes);
 
         return this;
     };
 
+    /**
+     * Injects a script into the container specified by `containerName`. In terms of position, the
+     * injected script is added directly after the script defined in `scriptAttribute`. The injected
+     * script is added as an inline script.
+     * @param scriptName
+     * @param containerName
```

Comment:
Agreed, I think you would call it after, but specifying the container seems very unDRY.

```
descript.add('urgent', {src: ['script1']});
descript.inject('urgent', {src: ['script1']}, function() { console.log('injection'); });
```

It seems to me like descript should be able to derive 'urgent' from the scriptAttribute that you are passing in. Or maybe it doesn't even need to if we setup our container data structures in a different way.